client/index.html

```diff
@@ -274,7 +274,7 @@ <h1 class="title">Settings</h1>
 								<label for="theme-select" class="sr-only">Theme</label>
 								<select id="theme-select" name="theme" class="input">
 									{{#each themes}}
-										<option value="{{filename}}">
+										<option value="{{url}}">
```

Comment:
Local themes don't have `plugins/` in their url. So basically after upgrading to this PR, everyone will have their theme reset, as `example` will change to `themes/example.css`.